sdcm/tester.py

```diff
@@ -163,6 +163,7 @@ def setUp(self):
         db_node_address = self.db_cluster.nodes[0].private_ip_address
         self.loaders.wait_for_init(db_node_address=db_node_address)
         nodes_monitored = [node.public_ip_address for node in self.db_cluster.nodes]
+        nodes_monitored += [node.public_ip_address for node in self.loaders.nodes]
```

Comment:
Interesting, I thought we only had provisions to monitor the database nodes. If we can also monitor loaders, that is cool.